cranelift/README.md

```diff
@@ -85,6 +84,10 @@ Building Cranelift
 Cranelift uses a [conventional Cargo build
 process](https://doc.rust-lang.org/cargo/guide/working-on-an-existing-project.html).
 
+Building cranelift only works with the nightly version of rust.
```

Comment:
Could we say something like: "Building all crates in the wasmtime + Cranelift repository will require a nightly version of Rust. With stable Rust, one can build the `wasmtime` binary (`cargo build` in the repository root) and the `clif-util` binary (`cargo build -p cranelift-tools`). Core Cranelift crates `cranelift-codegen`, `cranelift-wasm` and their dependencies will build on stable Rust."? In other words, I don't want to give the impression that nothing at all is usable without nightly.

Also should we say something about our goals? IIRC @alexcrichton had a related PR recently. Are we working toward only using stable Rust eventually or will we always have some nightly-only crates?

Reply:
Lightbeam was ported to stable in https://github.com/bytecodealliance/wasmtime/pull/2370. I don't think there were any other crates that require nightly.